include/sway/criteria.h

```diff
@@ -25,7 +25,9 @@ struct criteria {
 	pcre *instance;
 	pcre *con_mark;
 	uint32_t con_id; // internal ID
+  #ifdef HAVE_XWAYLAND
```

Comment:
Style:
1. We're using tabs, not spaces
2. We're not putting indentation for pre-processor directives